hibernate-core/src/main/java/org/hibernate/dialect/MySQLDialect.java

```diff
@@ -566,6 +567,11 @@ public JDBCException convert(SQLException sqlException, String message, String s
 		};
 	}
 
+	@Override
+	public NameQualifierSupport getNameQualifierSupport() {
+		return NameQualifierSupport.CATALOG;
```

Comment:
Copying comment from [initial commit](https://github.com/hibernate/hibernate-orm/commit/f4f4d49da0de10b96cb52829de2315722fb148eb):

Shouldn't it be NameQualifierSupport.SCHEMA and not CATALOG (since MySQL use the concept of schema and not the catalogs).
Whenever there would be an Entity class annotated with @Table(schema = "tech", name = "point"), the class QualifiedObjectNameFormatterStandardImpl would format the name into point and not the tech.point (since it does not define catalog)